purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -420,13 +414,7 @@ class Purchases internal constructor(
         storeProduct: StoreProduct,
         callback: PurchaseCallback
     ) {
-        val purchaseOption = storeProduct.bestPurchaseOption
```

Comment:
should we add a test checking that when no purchaseoption is passed in, we use the defaultoption? sorry if it already exists and i missed it..